src/__swaps__/screens/Swap/providers/swap-provider.tsx

```diff
@@ -132,28 +133,48 @@ interface SwapProviderProps {
   children: ReactNode;
 }
 
+const getInitialSliderXPosition = ({
+  inputAmount,
+  maxSwappableAmount,
+}: {
+  inputAmount: string | undefined;
+  maxSwappableAmount: string | undefined;
+}) => {
+  if (inputAmount && maxSwappableAmount) {
+    return clamp(+mulWorklet(divWorklet(inputAmount, maxSwappableAmount), SLIDER_WIDTH), 0, SLIDER_WIDTH);
```

Comment:
will `maxSwappableAmount` always be defined here? Could there be a case where it's undefined as declared in the types? Division by 0?

Reply:
that `if` should check for that